pkg/Library/Shader/ShaderNodes.js

```diff
@@ -7,7 +7,8 @@
 
 export const FragmentShader = {
   $meta: {
-    id: 'Fragment Shader',
+    id: 'FragmentShader',
```

Comment:
i discovered yesterday, that we actually keep the whitespaces in nodetype IDs... just saying..
maybe we should make the more restrictive? like `[a-zA-z0-9]+`?

Reply:
I think I'm consistent to suggest that most things should have 'name' and then we can _normalize_ into an 'id' format as a second stage. At that point we only use 'id' if we _really want_ that id, in which case we probably should let it be whatever.

Reply:
I didn't mean to jump over agreeing that `id` should probably follow JS identifier rules.